src/components/derivation/Create/index.tsx

```diff
@@ -31,6 +52,10 @@ function DerivationCreate() {
         setNewCollectionKey((k) => k + 1);
         resetTransformationCreateState();
         resetBindingState(undefined, true);
+
+        if (collectionsPrefilled) {
+            resetTableSelections();
+        }
```

Comment:
Unsure if preserving the selected collection table rows when the derivation create dialog is closes after the _New Transformation_ CTA initiated the workflow is a feature -- as it very well _could_ be -- so I wanted to keep that behavior intact.

Reply:
That is what I was thinking too. It is a stretch but it is possible.